### PR TITLE
Update to NServiceBus 6.0.0-beta006

### PR DESCRIPTION
**Pull request by MarcinHoppe**
@Particular/nservicebus-maintainers Please review and merge.

Connects to https://github.com/Particular/V6Launch/issues/65

### CODE REVIEW

File: src/NServiceBus.Callbacks.Tests/When_sending_reply_to_the_request.cs [hunk 1]
```diff
@@ -53,7 +52,7 @@ public void Below_v4_3_0_should_return_value_for_all_intents(string nsbVersion,
             Assert.IsTrue(result.HasValue);
         }
 
-        class IncomingMessage : Transports.IncomingMessage
+        class IncomingMessage : Transport.IncomingMessage
```
Comment (danielmarbach) [inline src/NServiceBus.Callbacks.Tests/When_sending_reply_to_the_request.cs]: Could we rename this to something like FakeIncomingMessage or PresetupIncomingMessage to remove the full qualification above?

File: src/NServiceBus.Callbacks.Tests/When_sending_reply_to_the_request.cs [hunk 2]
```diff
@@ -21,7 +20,7 @@ public void From_v4_3_0_should_return_value_only_for_reply_intent(string nsbVers
             var correlationId = new Guid().ToString();
             var lookup = new RequestResponseStateLookup();
             lookup.RegisterState(correlationId, new RequestResponseStateLookup.State());
-            Transports.IncomingMessage message = new IncomingMessage(nsbVersion, intent);
+            Transport.IncomingMessage message = new IncomingMessage(nsbVersion, intent);
```
Comment (danielmarbach) [inline src/NServiceBus.Callbacks.Tests/When_sending_reply_to_the_request.cs]: var

File: src/NServiceBus.Callbacks.Tests/When_sending_reply_to_the_request.cs [hunk 3]
```diff
@@ -44,7 +43,7 @@ public void Below_v4_3_0_should_return_value_for_all_intents(string nsbVersion,
...
```
Comment (danielmarbach) [inline src/NServiceBus.Callbacks.Tests/When_sending_reply_to_the_request.cs]: var

File: src/NServiceBus.Callbacks.AcceptanceTests/When_using_int_response_and_conventions.cs [hunk 4]
```diff
@@ -34,11 +34,11 @@ public class Replier : EndpointConfigurationBuilder
         {
             public Replier()
             {
-                EndpointSetup<DefaultServer>(busConfiguration =>
+                EndpointSetup<DefaultServer>(endpointConfiguration =>
```
Comment (danielmarbach) [inline src/NServiceBus.Callbacks.AcceptanceTests/When_using_int_response_and_conventions.cs]: just use `c` like in other tests

File: src/NServiceBus.Callbacks.AcceptanceTests/When_using_enum_response_and_conventions.cs [hunk 5]
```diff
@@ -41,11 +41,11 @@ public class Replier : EndpointConfigurationBuilder
         {
             public Replier()
             {
-                EndpointSetup<DefaultServer>(busConfiguration =>
+                EndpointSetup<DefaultServer>(endpointConfiguration =>
```
Comment (danielmarbach) [inline src/NServiceBus.Callbacks.AcceptanceTests/When_using_enum_response_and_conventions.cs]: just use `c` like in other tests

**Comment by danielmarbach** [comment]
@MarcinHoppe made a few comments

**Comment by MarcinHoppe** [comment]
@danielmarbach Remarks addressed. Build is green.

**Comment by danielmarbach** [comment]
@MarcinHoppe sorry I just figured we probably need a nuspec bump as well

**Comment by MarcinHoppe** [comment]
@danielmarbach Nuspec bumped.

@Particular/nservicebus-maintainers This is ready for merge.

**Pull request merged**